base/cvd/cuttlefish/host/commands/cvd/server_command/create.cpp

```diff
@@ -296,6 +315,43 @@ Result<selector::LocalInstanceGroup> CvdCreateCommandHandler::GetOrCreateGroup(
   return groups[0];
 }
 
+// For backward compatibility, we add extra symlink in home dir
+Result<void> CvdCreateCommandHandler::CreateSymlinks(
+    const selector::LocalInstanceGroup& group) {
+  auto system_wide_home = CF_EXPECT(SystemWideUserHome());
+  CF_EXPECT(EnsureDirectoryExists(group.HomeDir()));
+  auto smallest_id = std::numeric_limits<unsigned>::max();
+  for (const auto& instance : group.Instances()) {
+    // later on, we link cuttlefish_runtime to cuttlefish_runtime.smallest_id
+    smallest_id = std::min(smallest_id, instance.id());
+    const std::string instance_home_dir = fmt::format(
+        "{}/cuttlefish/instances/cvd-{}", group.HomeDir(), instance.id());
+    CF_EXPECT(EnsureSymlink(instance_home_dir,
+                            fmt::format("{}/cuttlefish_runtime.{}",
+                                        system_wide_home, instance.id())));
+    CF_EXPECT(EnsureSymlink(group.HomeDir() + "/cuttlefish",
+                            system_wide_home + "/cuttlefish"));
+  }
+  // The config file needs to be copied instead of symlinked because when the
+  // group is removed the original file will be deleted leaving the symlink
+  // dangling. The config file in the home directory is used by
+  // cvd_internal_start to persist the user's choice for
+  // -report_anonymous_usage_stats.
+  CF_EXPECT(
+      Copy(group.InstanceDir(group.Instances()[0]) + "/cuttlefish_config.json",
```

Comment:
The `group.Instances()[0]` access should be after the `CF_EXPECT_NE` check, as the `CF_EXPECT_NE` check proves that `group.Instances()` is not empty.

Reply:
Done